DotNetHowToMultipleDataSources/Program.cs

```diff
@@ -0,0 +1,153 @@
+                Environment.Exit(-1);
+            }
+
+            if (configuration["CosmosDBConnectionString"] == "Put your Cosmos DB connection string here")
+            {
+                Console.Error.WriteLine("Specify CosmosDBConnectionString in appsettings.json");
+                Environment.Exit(-1);
+            }
+
+            SearchServiceClient searchService = new SearchServiceClient(
+                searchServiceName: configuration["SearchServiceName"],
+                credentials: new SearchCredentials(configuration["SearchServiceAdminApiKey"]));
+
+            Console.WriteLine("Creating index...");
+            Index index = new Index(
+                name: "hotels",
+                fields: FieldBuilder.BuildForType<Hotel>());
+            // If we have run the sample before, this index will be populated
+            // We can clear the index by deleting it if it exists and creating
+            // it again
+            bool exists = await searchService.Indexes.ExistsAsync(index.Name);
+            if (exists)
+            {
+                await searchService.Indexes.DeleteAsync(index.Name);
+            }
+
+            await searchService.Indexes.CreateAsync(index);
+
+            Console.WriteLine("Creating data sources...");
+
+            // The SQL sample data set has a table name of "hotels"
+            // The SQL sample data set uses SQL integrated change tracking for change detection
+            // This means that when the SQL indexer runs, it will be able to detect which data has
+            // changed since the last run using built in change tracking
+            // See this link for more information
+            // https://docs.microsoft.com/sql/relational-databases/track-changes/about-change-tracking-sql-server
+            DataSource sqlDataSource = DataSource.AzureSql(
+                name: "azure-sql",
+                sqlConnectionString: configuration["AzureSQLConnectionString"],
+                tableOrViewName: "hotels");
+            sqlDataSource.DataChangeDetectionPolicy = new SqlIntegratedChangeTrackingPolicy();
+            // The SQL data source does not need to be deleted if it was already created,
+            // but the connection string may need to be updated if it was changed
+            await searchService.DataSources.CreateOrUpdateAsync(sqlDataSource);
+
+            // The JSON sample data set has a collection name of "hotels"
+            // The JSON sample data set uses Cosmos DB change tracking for change detection
+            // This means that when the Cosmos DB indexer runs, it will be able to detect which data has
+            // changed since the last run using built in change tracking
+            // See this link for more information
+            // https://docs.microsoft.com/azure/search/search-howto-index-cosmosdb#indexing-changed-documents
+            DataSource cosmosDbDataSource = DataSource.DocumentDb(
```

Comment:
@mgottein I thought we deprecated `DocumentDb` in the SDK (there should be a new `CosmosDb` method and the old method should give a compiler warning). Maybe we missed this. Can you check and file an issue if that's the case?

Reply:
Thanks!